Plugins/NativeInput/Source/Shared/NativeInput.cpp

```diff
@@ -111,6 +130,11 @@ namespace Babylon::Plugins
         PointerMove(MOUSE_POINTER_ID, x, y, DeviceType::Mouse);
     }
 
+    void NativeInput::Impl::MouseWheel(uint32_t wheelAxis, int32_t scrollValue)
+    {
+        PointerScroll(MOUSE_POINTER_ID, wheelAxis, scrollValue, DeviceType::Mouse);
```

Comment:
Is there such thing as "pointer scroll" for anything other than a mouse?

Reply:
Not really.  Closest thing would be a touchpad gesture but that would still register as a mouse wheel scroll

Reply:
Meh I guess either way, I was just unsure since it was parameterized but I wasn't sure if we'd ever have any other kind of pointer with scrolling.